code-example/code-example/SimpleArithmetic.cs

```diff
@@ -2,14 +2,19 @@
 {
     public class SimpleArithmetic
     {
-        public decimal Addition(int x, int y)
+        public int Addition(int x, int y)
         {
-            return x + y;
+            return y + x;
```

Comment:
Don't change the order.
```suggestion
            return x + y;
```